### PR TITLE
Converts static config account provider into dynamic calls to Clouddr…

### PR DESCRIPTION
**Pull request by ttomsu**
…iver.

@duftler @cfieber @jtk54 PTAL

@ajordens FYI

### CODE REVIEW

**Comment by duftler** [comment]
LGTM.

File: fiat-web/src/main/java/com/netflix/spinnaker/fiat/controllers/AuthorizeController.java [hunk 1]
```diff
@@ -35,6 +40,26 @@
   @Autowired
   private PermissionsRepository permissionsRepository;
 
+  @Value("${auth.getAll.enabled:false}")
+  @Setter
+  private boolean getAllEnabled;
+
+  @ApiModelProperty(value = "Used mostly for testing. Not really any real value to the rest of " +
```
Comment (duftler) [inline fiat-web/src/main/java/com/netflix/spinnaker/fiat/controllers/AuthorizeController.java]: I think this is the wrong type of annotation here, no?

Comment (ttomsu) [inline fiat-web/src/main/java/com/netflix/spinnaker/fiat/controllers/AuthorizeController.java]: Fixed.

**Comment by jtk54** [comment]
LGTMA

**Pull request merged**